plugins/engines/ospray/OSPRayScene.cpp

```diff
@@ -110,6 +110,11 @@ void OSPRayScene::commit()
                 BRAYNS_INFO << "Committing " << modelDescriptor->getName()
                             << std::endl;
                 _addInstance(_rootModel, impl.getModel(), transformation);
+
+#ifndef OSPRAY_SUPPORTS_SUBMODEL_INSTANCES
```

Comment:
From my understanding, OSPRAY_SUPPORTS_SUBMODEL_INSTANCES will apparently never be true :( Do you still want to keep it?

Reply:
After I read the ticket of course not :) I'm about to push that.